tests/test_tools.py

```diff
@@ -101,7 +244,7 @@ def test_sum():
     r2 = X**2 + Y**2
     P = r2**-1.0
     ave, coord = angular_average(P, np.sqrt(r2), bins=20, bin_ave=False, average=False)
-    assert np.sum(P[r2 < 18.0]) == np.sum(ave)
+    assert np.sum(P[r2 < 9.0]) == np.sum(ave)
```

Comment:
why did this have to change?

Reply:
Because we changed the maximum bin to be the largest inscribed circle rather than the corner (i.e. your first comment around tools.py/line 20.